test/jdk/java/awt/Focus/6981400/Test1.java

```diff
@@ -156,24 +173,21 @@ public static void test(Component compToClick) {
         tracking = true;
 
         robot.keyPress(KeyEvent.VK_TAB);
-        robot.delay(50);
         robot.keyRelease(KeyEvent.VK_TAB);
-        robot.delay(50);
 
         robot.keyPress(KeyEvent.VK_TAB);
-        robot.delay(50);
         robot.keyRelease(KeyEvent.VK_TAB);
-        robot.delay(50);
 
         robot.keyPress(KeyEvent.VK_TAB);
-        robot.delay(50);
         robot.keyRelease(KeyEvent.VK_TAB);
 
-        robot.delay(50);
         Util.clickOnComp(compToClick, robot);
 
-        robot.delay(50);
-        Util.clickOnTitle(f0, robot);
+        if (Platform.isOnWayland()) {
```

Comment:
Do we really need an additional click on the title in this case?
Just `toFront` should be enough on all platforms.

Reply:
`toFront()`  is known to be broken on some versions of metacity/gnome, not sure it was fixed or not, that was a reason why the "click" workaround was introduced. If it works fine on Wayland we can just use it and leave the old x11 workaround as is.

Reply:
Update to `"toFront + click on title" on X11 and just "toFront" on Wayland`

> not sure it was fixed or not

Looks like so, I also tested just `toFront()` without the `clickonTitle()`, it passes on all platforms. But I kept the `clickOnTitle` just in case.